manifests/ha/base/redis-ha/chart/values.yaml

```diff
@@ -9,7 +9,7 @@ redis-ha:
   haproxy:
     enabled: true
     image:
-      tag: 2.0.29-alpine
+      tag: 2.5.7-alpine
```

Comment:
Upon further investigation of [HAProxy's release cycle](https://www.haproxy.org), stable version 2.5 is EOL 2023-Q1.  We should choose either 2.6(EOL 2027-Q2) or 2.4(EOL 2026-Q2) both are LTS.

Should we pin a version like we do for image builds and set the imagePullPolicy to Always? A 2.x version will always point to the the latest patch version for HAProxy.

i.e.
```yaml
 haproxy:
    enabled: true
    image:
      tag: 2.6-alpine
      pullPolicy: Always
```

Reply:
> Should we pin a version like we do for image builds and set the imagePullPolicy to Always? A 2.x version will always point to the the latest patch version for HAProxy.

Personally I like setting the whole version, making the runtime behavior of an Argo CD install more deterministic. But I could see a case either way.

> We should choose either 2.6(EOL 2027-Q2) or 2.4(EOL 2026-Q2) both are LTS.

I wonder if we should just use the most recent version of the chart which pointed at the 2.4.x series (to minimize chart/image incompatibility risk). If you're not opposed it, that's the route I'll go.

Reply:
@34fathombelow I'll wait for your PR to be merged and then update this one.